js/libs/form.js

```diff
@@ -227,24 +227,9 @@ Fliplet().then(function() {
       });
     }
 
-    function saveProgress() {
```

Comment:
This was only just recently added to fix an issue. See PR: https://github.com/Fliplet/fliplet-widget-form-builder/pull/514

Can you justify why you're moving this again, and have you checked it doesn't reintroduce the bug?